--- content/standalone.sh	2020-06-03 21:14:32.765149040 +0200
+++ expected/standalone.sh	2020-06-03 21:16:26.037265392 +0200
@@ -21,6 +21,12 @@
 echo ""
 echo "  JAVA_OPTS: $JAVA_OPTS"
 echo ""
+echo "  JAVA_TOOL_OPTIONS: $JAVA_TOOL_OPTIONS"
+echo ""
+echo "  _JAVA_OPTIONS: $_JAVA_OPTIONS"
+echo ""
+echo "  IBM_JAVA_OPTIONS: $IBM_JAVA_OPTIONS"
+echo ""
 echo "========================================================================="
 echo ""
 
